packages/api/src/schema/schema.yaml

```diff
@@ -908,6 +909,7 @@ components:
           type: string
           example: 39bf51c4-38eb-4e71-8d29-36da22035dc2
           description: UUID of Recording Session created by Mist
+          index: true
```

Comment:
Oh just realized one thing... We have turned off the automatic index creation on most tables but the VOD ones as they were locking the DB. I think it still makes sense to have the index declared here for the future, but we might need to create these indexes manually before deploying. 

Can you make sure to ping me once we merge this PR / plan on deploy to prod so I can help with that? A table scan on the `streams` table could be pretty catastrophic 🥲

Reply:
👍